internal/worker/logging.go

```diff
@@ -33,6 +35,10 @@ func LogDynamicAnalysisError(pkg *pkgmanager.Pkg, errorPhase analysisrun.Dynamic
 		"name", pkg.Name(),
 		"version", pkg.Version(),
 		"error", err)
+
+	if exitErr, ok := err.(*exec.ExitError); ok {
+		log.Debug("Command stderr", "stderr", exitErr.Stderr)
```

Comment:
Actually, I think https://pkg.go.dev/errors#example-As is the better way to do this is in newer Go?

Reply:
Thanks, I'll change it